platform/android/detect.py

```diff
@@ -169,7 +169,7 @@ def mySpawn(sh, escape, cmd, args, env):
     elif (sys.platform.startswith("darwin")):
         host_subpath = "darwin-x86_64"
     elif (sys.platform.startswith('win')):
-        if (platform.machine().endswith('64')):
+        if (env["arch"].endswith('64')):
```

Comment:
I'm not sure that correct, this code chunk seems to be specifically about the host platform, not the target (which `env["arch"]` is).

Reply:
Indeed, my bad :sweat_smile: